store/src/java/com/zimbra/cs/html/owasp/HtmlElementsBuilder.java

```diff
@@ -0,0 +1,54 @@
+package com.zimbra.cs.html.owasp;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+import org.owasp.html.ElementPolicy;
+
+/*
```

Comment:
nit: copyright before imports?  That would seem like our typical pattern and consistency is good.